### PR TITLE
Add ansible for gnome messsage banner UBTU-20-010003

### PR DESCRIPTION
**Pull request by dexterle**
#### Description:

- Fix UBTU-20-010003
- Add Ansible remediation for Ubuntu

#### Rationale:

- Part of Ubuntu 20.04 DISA STIG v1r9 profile upgrade
- https://github.com/ComplianceAsCode/content/pull/10738

#### Review Hints:

Build the product:
```
./build_product ubuntu2004
```
To test these changes with Ansible:
```
ansible-playbook build/ansible/ubuntu2004-playbook-stig.yml --tags "DISA-STIG-UBTU-20-010003"
```

Checkout Manual STIG OVAL definitions, and use software like DISA STIG Viewer to view definitions.
```
git checkout dexterle:add-manual-stig-ubtu-20-v1r9
```

This STIG can not be tested with the latest Ubuntu 2004 Benchmark SCAP. Please perform a manual check given the check text. For reference, please review the latest artifacts: https://public.cyber.mil/stigs/downloads/

### CODE REVIEW

**Review by dodys — CHANGES_REQUESTED** [review]
please check bash remediation and add the bit of editing dconf database

File: linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_login_banner_text/ansible/ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+{{{ ansible_instantiate_variables("login_banner_text") }}}
+
+- name: "{{{ rule_title }}}"
```
Comment (dexterle) [inline linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_login_banner_text/ansible/ubuntu.yml:8]: Use a more descriptive name, follow the style guide

**Comment by jan-cerny** [comment]
ping

**Comment by dodys** [comment]
dexterle reached out last week about continuing work on this PRs, and I've asked to open new ones for a clean state. Therefore closing this one.

**Pull request closed without merging**